### PR TITLE
update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by timbussmann**
After going through the pain today, I think it would make sense to document the required steps and settings in the repositories?

Something else to add?

@Particular/azure-maintainers what do you think?

### CODE REVIEW

**Comment by melkio** [comment]
@timbussmann :+1: 
It could be useful for ASB repo too / @SeanFeldman @yvesgoeleven

**Comment by timbussmann** [comment]
> It could be useful for ASB repo too / @SeanFeldman @yvesgoeleven 

I agree :+1: Maybe someone else who knows about the details (there is no emulator, right?) can "port" that over to ASB.

**Comment by melkio** [comment]
Ok @timbussmann. I'll raise a new issue on ASB repo to copy&adapt your document (no emulator and a "new" variable to configure)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -30,6 +30,13 @@ top of things.
 * We have a resharper layer that applies our coding standards so make sure that you're "all green in reshaper"
 
 
+## Running the Acceptance Tests
+
+Follow these steps to run the acceptance tests locally:
+* Add a new environment variable `Transport.UseSpecific` with the value `AzureStorageQueueTransport`
+* Add a new environment variable `AzureStorageQueueTransport.ConnectionString` containing a connection string to your Azure storage account or use use `UseDevelopmentStorage=true` to use the [Azure Storage Emulator](https://azure.microsoft.com/en-us/documentation/articles/storage-use-emulator/) (make sure to start it before you run the tests).
```
Comment (SeanFeldman) [inline CONTRIBUTING.md]: " or use use" => "or use"

Comment (timbussmann) [inline CONTRIBUTING.md]: :+1: fixed

**Comment by Scooletz** [comment]
All good

**Pull request merged**